db/migrate/20161124135413_create_grants_voters.rb

```diff
@@ -0,0 +1,9 @@
+class CreateGrantsVoters < ActiveRecord::Migration
+  def change
+    create_table :grants_voters do |t|
```

Comment:
I tried to do this as a join table but join tables don't have primary keys so it was impossible to delete single records.